index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav-bar">
+      <span class="name"><a href="#">Miguel</a></span>
+      <span class="menu-icon"><a href="#"><img src="images/menu.svg" alt="menu icon">
+        </a></span>
+    </nav>
+  </header>
+  <section class="container">
+    <h2 class="greetings">Hey There. I’m Miguel</h2>
```

Comment:
[Optional]

- [ ] It's better if you make this element an `h1` element instead of `h2` for semantics and to improve SEO. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.